### PR TITLE
Bug 2122521: Unique IP address validation for IBM storage system

### PR DESCRIPTION
**Pull request by alonfirestein**

### CODE REVIEW

**Comment by SanjalKatiyar** [comment]
ok-to-test

**Comment by SanjalKatiyar** [comment]
@bipuladh LGTM.
can u enable workflows and take a look once at the PR as well...
https://github.com/red-hat-storage/odf-console/issues/879 issue for updating and refactoring later...

File: packages/odf/components/create-storage-system/create-storage-system-steps/create-storage-class-step/create-storage-class-step.tsx [hunk 1]
```diff
@@ -52,12 +60,37 @@ export const CreateStorageClass: React.FC<CreateStorageClassProps> = ({
...
+  const dataLoaded = flashSystemLoaded && secretLoaded;
+  const dataLoadError = flashSystemLoadError || secretLoadError;
 
   const { schema, fieldRequirements } = React.useMemo(() => {
     const existingNames =
-      loaded && !loadError ? data?.map((dataItem) => getName(dataItem)) : [];
+      scLoaded && !scLoadError ? scData?.map((data) => getName(data)) : [];
+
+    const existingFlashSystemSecretNames =
+      flashSystemLoaded && !flashSystemLoadError
```
Comment (SanjalKatiyar) [inline packages/odf/components/create-storage-system/create-storage-system-steps/create-storage-class-step/create-storage-class-step.tsx:81]:
```suggestion
      dataLoaded && !dataLoadError
```

**Comment by SanjalKatiyar** [comment]
@alonfirestein plz squash the commits into a single commit. Also, if u notice second commit is showing me as a co-author, seems like there is some issue and commit is not clean.

**Comment by bvered** [comment]
@SanjalKatiyar please see email from Alon , please assist squash commits as you requested.

**Pull request closed without merging**